plover/assets/dict_unsorted.json

```diff
@@ -121866,7 +121866,6 @@
 "KWRA/AUL": "y'all",
 "KWRAOU/AUL": "y'all",
 "KWRAB/PWA/TKAB/PWA": "yabba dabba",
-"KWRAT": "yacht",
```

Comment:
There's no entry for "@" in my release version of the dictionaries ... I had to add it to my personal dictionary. Might be a good idea to add it, though, maybe use "KRWA_Asterisk_T" to leave yacht as is?